checker/check_result.go

```diff
@@ -85,6 +85,45 @@ type CheckResult struct {
 	Reason   string        `json:"-"` // A sentence describing the check result (score, etc)
 }
 
+// CreateProportionalScore() creates a proportional score.
+func CreateProportionalScore(b, t int) int {
```

Comment:
ignore this file except the `AggregateScoresWithWeight` function. Will disappear from the PR once https://github.com/ossf/scorecard/pull/739 is merged.